ui/components/app/deposit-popover/on-ramp-item.js

```diff
@@ -17,35 +23,49 @@ const OnRampItem = ({
     return null;
   }
   return (
-    <Box paddingRight={6} paddingLeft={6}>
+    <Box className="on-ramp-item" as="li" marginRight={6} marginLeft={6}>
       <Box
         paddingTop={6}
         paddingBottom={6}
-        style={{
-          borderBottomSize: '1px',
-          borderBottomColor: COLORS.BORDER_MUTED,
-        }}
+        display={DISPLAY.FLEX}
+        alignItems={ALIGN_ITEMS.CENTER}
+        flexWrap={FLEX_WRAP.WRAP}
       >
-        <Box width={FRACTIONS.HALF}>{logo}</Box>
-        <Typography
-          variant="h6"
-          fontWeight="bold"
-          boxProps={{
-            paddingTop: 2,
-            paddingBottom: 2,
-          }}
+        <Box
+          width={[
+            BLOCK_SIZES.ONE_THIRD,
+            BLOCK_SIZES.ONE_THIRD,
+            BLOCK_SIZES.ONE_FIFTH,
+          ]}
+          display={DISPLAY.FLEX}
+          justifyContent={[
+            JUSTIFY_CONTENT.FLEX_START,
+            JUSTIFY_CONTENT.FLEX_START,
+            JUSTIFY_CONTENT.CENTER,
+          ]}
+          paddingLeft={[0, 4]}
+          paddingRight={[0, 4]}
```

Comment:
This is to fix too much padding on md screens
```suggestion
          paddingLeft={[0, 0, 4]}
          paddingRight={[0, 0, 4]}
```